src/marqo/tensor_search/backend.py

```diff
@@ -94,21 +95,13 @@ def add_customer_field_properties(config: Config, index_name: str,
                         utils.generate_vector_name(field_name[0])): {
                         "type": "knn_vector",
                         "dimension": model_properties["dimensions"],
-                        "method": {
```

Comment:
this could be `"ann_method"` for clarity, since it's handling ann parameters. but may not be important as this isn't exposed to users

Reply:
IIRC, this is explicitly defined by Opensearch, we must follow their convention.